known-issues-vs2017.md

```diff
@@ -164,117 +135,74 @@ If the DataContext class has a member (property, method, variable) defined with
-
-* #### Issue:
-If you try to publish an ASP.NET Core Web Application (.NET Framework), you will run into the following error: "DestinationFiles" refers to 1 item(s), and "SourceFiles" refers to 2 item(s). They must have the same number of items
-
-* #### Workaround:
-None available
-
-### Publish crashes
-Publish crashes on locales that do not use '.' as a decimal separator
-
-* #### Issue:
-A bug in publishing fails to distinguish decimal separators in a language-neutral way.
-
-* #### Workaround:
-Before publishing, set VS locale to ENU.
-
-### Cannot configure web server settings
-Property pages of ASP.NET Core projects do not allow you to configure Web Server Settings
-
-* #### Issue:
-Web Server settings such as App URL, ability to enable SSL, Windows Authentication are not available in the property pages of an ASP.NET Core project
+```
+            var host = new WebHostBuilder()
+                .UseKestrel()
+                .UseContentRoot(Directory.GetCurrentDirectory())
+                .UseIISIntegration()
+                .UseStartup<Startup>()
+                .UseApplicationInsights()
+                .Build();
+```
 
-* #### Workaround:
-You can edit the launchSettings.json file and apply settings similar to .NET Core tooling in Visual Studio 2015
+### Docker enabled project fails to build
 
-### Razor IntelliSense not working
-Razor IntelliSense Completion issues in .NET Core projects
+* #### Issue:  
+When creating an ASP.NET Core project, if you check the "Enable Container (Docker) Support" checkbox but do not have "Docker for Windows" installed, the created project will fail to build with the following error: 
 
-* #### Issue:
-Completion at the end of razor expressions doesn't work well in RC. eg, typing "@DateTi." or "@DateTime." (w/o the quotes) will not commit properly, and may mark the dot as markup.  Please report any additional IntelliSense Issues using "Send Feedback."
+  Microsoft.DotNet.Docker.CommandLineClientException: 'docker-compose' was not found. Please verify that it is available on %PATH%, or for troubleshooting, follow instructions from http://aka.ms/DockerToolsTroubleshooting 
 
-* #### Workaround:
-Backspace one character and retry
+* #### Workaround: 
+Install [Docker for Windows](https://docs.docker.com/docker-for-windows/) to resolve the issue
 
-### Scaffolded files not included in project
-Scaffolded files may not be included in user's project in certain cases, where user's project defines exclusions.
+### No suggestions to install missing packages
 
-* #### Issue:  
-If user project has a globbing pattern for ```<Compile>``` item group with a exclude pattern. And if the files generated by scaffolding match the pattern, they are not included in the project. For example: If project has ```<Compile Include = "**/*.cs" Exclude="ExcludedDir/*.cs" />``` and scaffolding generates a file in 'DefaultController.cs' ExcludedDir, it will not be included in the project for compilation. 
+* #### Issue: 
```

Comment:
Move this issue near the top of the ASP.NET Core list as it is a commonly requested feature